integreat_cms/cms/views/utils/hix.py

```diff
@@ -47,6 +49,19 @@ def lookup_hix_score_helper(text: str) -> dict:
     :param text: The text to calculate the hix score for
     :return: The score for the given text
     """
+    # If the html has no text, don't send a request to textlab and immediately return the `None` score,
```

Comment:
I was taught that comments should say _what_ code is doing but instead _why_ it's are doing something. Also in this particular case I think your code is quite understandable, therefore I would either delete the comment all together or change it to answer why we don't want to send empty/None requests to TextLab :)

Reply:
I've added a test case that is better suited for documentation purposes and, more importantly, helps safeguard us from regressions. :)